src/main/frontend/components/command_palette.css

```diff
@@ -1,4 +1,4 @@
-.cp__palette {
+.cp__palette, .cp__select {
```

Comment:
I just reused the existing css for now to avoid duplication. We could also copy it if we prefer